README.rst

```diff
@@ -87,6 +87,19 @@ Installation
 
 The library is Python 3 only!
 
+PyPI contains binary wheels for Linux, Windows and MacOS.  If you want to install
+``yarl`` on another operation system (or *Alpine Linux* inside a Docker) the
+Tarball will be used to compile the library from sources.  It requires C compiler and
+Python headers installed.
+
+To skip the compilation please use `YARL_NO_EXTENSIONS` environment variable,
+e.g.:
+
+.. code-block:: bash
+
+   $ YARL_NO_EXTENSIONS=1 pip install yarl
+
+Please note, Pure Python (uncompiled) version is much slower.
```

Comment:
Please also mention that it's always pure-python under PyPy interpreter series.

Reply:
I'm not sure that PyPy should be explicitly noted.
The performance under PyPy is fine; PyPy executes C Extensions slower than native Python code.

Reply:
Just to inform users that they don't have to set `YARL_NO_EXTENSIONS` as it won't change anything.

Reply:
"Please note Pure Python (uncompiled) version is much slower. However, PyPy always uses pure python implementation, and, as such, is unaffected by this variable."

Would something like this work?

Reply:
s/is/it is/